resources/user.py

```diff
@@ -74,8 +77,20 @@ def patch(self,user_id):
         if not user:
             return {"Message": "Unable to find user."}, 400
         
+        # user.role = RoleEnum(data['role'])
```

Comment:
Is this commented out code needed later?

Reply:
Thanks, somehow missed it during cleanup